server/sonar-ce/src/main/java/org/sonar/ce/configuration/CeConfiguration.java

```diff
@@ -22,7 +22,12 @@
 public interface CeConfiguration {
 
   /**
-   * The number of workers to process CeTasks concurrently.
+   * The number of worker threads, integer strictly greater than 0.
+   */
+  int getWorkerThreadCount();
```

Comment:
Maybe we should different names for the 2 methods, as it sounds too close for me.
- `getMaxWorkerCount`
- `getCurrentWorkerCount`
?